modules/custom/az_person/config/install/core.entity_form_display.node.az_person.default.yml

```
@@ -151,6 +152,14 @@ content:
     third_party_settings:
       media_library_edit:
         show_edit: '1'
+  field_az_netid:
+    type: string_textfield
+    weight: 6
+    region: content
+    settings:
+      size: 60
```

Thanks @joshuasosa.  We tested this in our 2024-07-24 PR meeting we tested this and found that using a size of 16 did not allow 16 characters to always fit so we're going to change this to 30.

```suggestion
      size: 30
```